templates/web.yml.template

```diff
@@ -29,7 +29,7 @@ zenodoUrl: {{ ZENODO_URL }}
 orcidClientID: {{ ORCID_CLIENT_ID }}
 orcidClientSecret: {{ ORCID_CLIENT_SECRET }}
 
-gitHubAppPrivateKeyFile: /home/dockstore_github_app_private_key.pem
+gitHubAppPrivateKeyFile: /home/github-key/dockstore-github-private-key.pem
```

Comment:
The way this reads, you have a user named `github-key` and the only reason they exist is so their home directory can contain a `.pem` key. Here, `/home/` is being used as a home directory, but the home directory should be `/home/<username>`.

Reply:
I changed this to the `/dockstore/github-key` directory: https://github.com/dockstore/compose_setup/pull/225/commits/587358c473dc0683dd9c8c9c8cd497dfc7c2c5e6